tests/test_openclhpp.cpp

```diff
@@ -3626,7 +3630,63 @@ void testDevice_GetInfo_CLDeviceName()
     clGetDeviceInfo_StubWithCallback(clGetInfo_testDeviceGetInfoCLDeviceName);
     cl::string deviceName = devicePool[0].getInfo<CL_DEVICE_NAME>();
     TEST_ASSERT_EQUAL_STRING(expected.c_str(), deviceName.c_str());
-} 
+}
+
+#if defined(cl_ext_device_fission)
+static cl_int clCreateSubDevices_testDevice_createSubDevices(
+    cl_device_id in_device, const cl_device_partition_property *properties,
+    cl_uint num_devices, cl_device_id *out_devices, cl_uint *num_devices_ret,
+    int num) {
+    return 0;
+}
+
+static cl_int clCreateSubDevicesEXT_testDevice_createSubDevices(
+    cl_device_id device_in, const cl_device_partition_property_ext *properties,
+    cl_uint n, cl_device_id *out_devices, cl_uint *num, int num_calls) {
+  cl_int ret = CL_SUCCESS;
+
+  TEST_ASSERT_EQUAL(CL_DEVICE_PARTITION_TYPES_EXT, *properties);
```

Comment:
This is OK for a unit test but do note that `CL_DEVICE_PARTITION_TYPES_EXT` is not a valid partition property according to the cl_ext_device_fission [spec](https://registry.khronos.org/OpenCL/extensions/ext/cl_ext_device_fission.txt).  If we wanted this to be a more real-world test case then we should choose different partition properties, such as perhaps `CL_DEVICE_PARTITION_EQUALLY_EXT` or `CL_DEVICE_PARTITION_BY_AFFINITY_DOMAIN_EXT`.

Reply:
Changed to proper value.